lib/volume_cache.js

```diff
@@ -222,4 +222,17 @@ export default class VolumeCache {
       this.cache[cacheName][instanceId][key] = value[key];
     }
   }
+
+  /**
+  Get the name and location of a cache on disk.
+
+  @param {String} Cache key in the format of <cache name>::<instance id>
+  */
+  splitKey(cacheKey) {
```

Comment:
nit: the naming of this does not really represent what it's doing.  Perhaps getCacheDetails or something.